osgpkitools/cert_request.py

```diff
@@ -110,6 +112,6 @@ def main():
 
     for fqdns in fqdns_list:
         print(f"Writing CSR for {fqdns[0]}...")
-        csr_obj = cert_utils.Csr(fqdns[0], output_dir=os.path.abspath(args.write_directory), altnames=fqdns[1:], location=loc)
+        csr_obj = cert_utils.Csr(fqdns[0], output_dir=os.path.abspath(args.write_directory), altnames=fqdns[1:], location=loc, key_length=args.key_length)
```

Comment:
```suggestion
        csr_obj = cert_utils.Csr(fqdns[0],
                                 output_dir=os.path.abspath(args.write_directory),
                                 altnames=fqdns[1:],
                                 location=loc,
                                 key_length=args.key_length)
```

We prefer to work with 120 char wide lines. I thought we had this documented at one point but it appears that we do not :(